src/ui.form-date-picker-range/configs/default.config.js

```diff
@@ -34,8 +34,8 @@ export default /*tw*/ {
   },
   periodsRow: "mb-1 flex min-w-64 gap-1",
   periodButton: `
-    flex h-[3.125rem] w-full cursor-pointer flex-col items-center justify-center rounded-lg bg-gray-100
-    px-1.5 py-2.5 text-center text-xs font-medium text-brand-900 hover:bg-brand-200
+    flex h-[3.125rem] w-full flex-col items-center justify-center bg-gray-100
+    px-1.5 py-2.5 text-xs font-medium hover:!bg-brand-200
```

Comment:
* set `size="xs"` and `square` after you can remove `px-1.5 py-2.5 text-xs`
* remove `font-medium`
* `bg-gray-100` - lets remove it and keep brand default
* keep default hover - remove this: `hover:!bg-brand-200`

So the final classes string should be: `h-[3.125rem] w-full flex flex-col items-center justify-center`